src/client/themes/explorer/index.ts

```diff
@@ -66,18 +70,17 @@ const explorerTheme: DefaultTheme = {
       },
     },
 
-    walletButton: {
-      background: '#212121',
-      color: '#eeeeee',
-    },
+    // walletButton: {
+    //   background: '#212121',
+    //   color: '#eeeeee',
+    // },
```

Comment:
Remove?

Reply:
Removed in 37090db0d7efc2706a2f90189ac28479f41cded6